examples/rh-sso/client-registrations/webhook_handler.lua

```diff
@@ -0,0 +1,105 @@
+local redis_pool = require 'client-registrations/redis_pool'
+local lom = require 'lxp.lom'
+local xpath = require 'luaxpath'
+local cjson = require 'cjson'
+
+local _M = {}
+
+local initial_access_token = "CHANGE_ME_INITIAL_ACCESS_TOKEN"
```

Comment:
IMHO our examples should not suggest editing them but rather provide means to inject the correct value like through environment variable.